config/machine_readable/how-to-vote.yml

```diff
@@ -34,12 +34,11 @@ faqs:
 
   - question: When you can vote
     answer: >
-      <p>Polling stations are open from 7am to 10pm on the day of an election ('polling day').</p>
+      <p>Polling stations are open from 7am to 10pm on 6 May ('polling day').</p>
```

Comment:
`Polling stations will be open from 7am to 10pm on 6 May (‘polling day’).`

Reply:
Good spot, thank you!